### PR TITLE
finished GET

### PR DESCRIPTION
**Pull request by Marian2910**

### CODE REVIEW

**Comment by maiernoema** [comment]
name of the packages should be in lowercase.

**Comment by maiernoema** [comment]
target files should not be visible every time when you push your changes.

File: src/main/java/com/Endava/EventTix/Service/OrdersService.java [hunk 1]
```diff
@@ -1,20 +1,64 @@
...
-    public List<Orders> getOrdersByCustomerId(int customerId) {
-        return ordersRepository.findOrdersByCustomerID_CustomerID(customerId);
+    public List<OrdersDTO> getOrdersByCustomerId(int customerId) {
+
+        List<OrdersDTO> ordersDTOList = new ArrayList<>();
+        List<Orders> orders = ordersRepository.findOrdersByCustomerID_CustomerID(customerId);
+
+        for (Orders order : orders) {
+            OrdersDTO ordersDTO = new OrdersDTO();
+            ordersDTO.setEventID(ticketCategoryRepository.findTicketCategoryByTicketCategoryId(order.getTicketCategoryID().getTicketCategoryId()).getEventID().getEventID());
```
Comment (maiernoema) [inline src/main/java/com/Endava/EventTix/Service/OrdersService.java]: Here we should have added some conditions but because you had to do a little application and your DB is already populated with valid data... it's fine.
However it's good to know in the future :D , i'll give you an example: if **order.getTicketCategorIDy()** is null, the following **order.getTicketCategoryID().getTicketCategoryId(**) will throw a **NPE**(null pointer exception). It is a valid explanation for all the cases from here.

File: src/main/java/com/Endava/EventTix/Controller/EventController.java [hunk 2]
```diff
@@ -1,35 +1,60 @@
...
+    @Autowired
+    public EventController(EventService eventService) {
+        this.eventService = eventService;
+    }
 
     @GetMapping
-    public List<Event> getEventsByVenueIdAndEventType(@RequestParam Integer venueID, @RequestParam String eventType) {
-        return eventService.getEventyByVenue(venueID, eventType);
-    }
+    public List<EventDTO> getEventsByVenueIdAndEventType(@RequestParam Integer venueID, @RequestParam String eventType) {
```
Comment (maiernoema) [inline src/main/java/com/Endava/EventTix/Controller/EventController.java]: why did you move the logic from service to controller?

File: src/main/java/com/Endava/EventTix/Controller/OrdersController.java [hunk 3]
```diff
@@ -20,21 +19,24 @@ public OrdersController(OrdersService ordersService) {
     }
 
     @GetMapping
-    public List<Orders> getOrders() {
-        return ordersService.getOrders();
+    public List<OrdersDTO> getEventsByVenueIdAndEventType() {
```
Comment (maiernoema) [inline src/main/java/com/Endava/EventTix/Controller/OrdersController.java]: why did you move the logic from service to controller?

File: src/main/java/com/Endava/EventTix/Controller/EventController.java (hunk 2 above)
Comment (Marian2910) [reply src/main/java/com/Endava/EventTix/Controller/EventController.java]: I moved the logic from controller to service in my last commit, in this stage of my project all the logic is in service.

**Review by maiernoema — APPROVED** [review]
